src/pages/admin/MySettings/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { Fragment, useRef } from 'react';
+import { Tab } from '@headlessui/react';
+import Breadcrumb from '../../components/shared/Breadcrumb';
+import AdminProfileTab from './Profile';
+import AdminAccountTab from './Account';
+
+const classNames = (...classes) => classes.filter(Boolean).join(' ');
+
+const TabNames = ['Profile', 'Account'];
+
+const AdminMySettings = () => {
+  const fileInput = useRef(null);
+  return (
+    <Fragment>
+      <div className='mb-5 flex text-brown-900'>
+        <Breadcrumb text='Dashboard' to='/admin/dashboard' />
+        <Breadcrumb text='My Settings' last />
+      </div>
+      <section>
+        <div className='text-2xl font-bold text-brown-900'>My Settings</div>
+        <hr className='mt-3 border border-orange-900' />
+        <div className='mx-8 mt-8 flex'>
+          <div className='flex flex-col items-center'>
+            <img
+              src='/images/Test05.webp'
+              className='mb-10 h-[294px] w-[294px] rounded-full'
+              alt='profile'
+            />
+            <div>
+              <input type='file' name='image' ref={fileInput} style={{ display: 'none' }} />
+              <button
```

Comment:
```suggestion
              <Button
                IsCustomizable= {true}
                className='rounded-md bg-black px-5 py-2 font-semibold text-white'
                onClick={() => fileInput.current.click()}
                buttonLabel = 'Upload Photo'
              />
```
something like the suggestion in the top. 
Please use the shared button component and pass a style prop.
we can create switch case in  the Button component to identify the default button and the customize one.
please pass also the onclick attribute in the button component as a prop